.github/workflows/emscripten.yml

```diff
@@ -58,3 +70,12 @@ jobs:
           pip install matplotlib pytest
           python -c "import pywt; print(pywt.__version__)"
           pytest --pyargs pywt
+
+        # https://anaconda.org/scientific-python-nightly-wheels/pywavelets
+        # WARNING: this job will overwrite existing wheels.
+      - name: Push to Anaconda PyPI index
+        if: (github.repository == 'PyWavelets/pywt') && (github.event_name == 'push' && github.ref == 'refs/heads/master') || (github.event_name == 'workflow_dispatch' && github.event.inputs.push_wheels == 'true') || (github.event_name == 'schedule')
+        uses: scientific-python/upload-nightly-action@b67d7fcc0396e1128a474d1ab2b48aa94680f9fc # v0.5.0
```

Comment:
I think getting Dependabot from #708 would be nice to keep this Action's version up to date.

Reply:
I would highly recommend it. Using groups and scoping to GitHub Actions makes it so that at most you are dealing with 1 PR per time interval that you've set.

c.f. https://github.com/h5py/h5py/pull/2384